.eslintrc.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
This file is also referenced in .gitignore. Was checking in the file intentional?